src/containers/views/DocumentEdit.js

```diff
@@ -113,6 +114,8 @@ export class DocumentEdit extends Component {
     } = currentDocument;
     const [directory, ...rest] = params.splat;
 
+    const metafields = injectDefaultFields(config, directory, collection, front_matter);
```

Comment:
`config` is since recently, 
```
{"content": {"foo": "bar"}, "raw_content": "foo: bar\n"}
```
So **either** this line and all instances of `config` passed to `injectDefaultFields` should be changed to `config.content` **or** change `injectDefaultFields` function to extract `defaults` from `config.content`